File: frontend/src/types/standings.ts
// a Standing includes some details of its team
export interface Standing {
  team: {
    id: number;
    name: string;
    short_name: string;
    logo_url: string;
  };
  position: number;
  matches_played: number;
  wins: number;
  draws: number;
  losses: number;
  goals_for: number;
  goals_against: number;
  goal_difference: number;
  points: number;
}

File: backend/app/schemas/standings.py
from pydantic import BaseModel

from app.schemas.common import TeamStandingsSummary, TeamSummary


class StandingResponse(TeamStandingsSummary):
    team: TeamSummary


class StandingRefreshRow(BaseModel):
    external_team_id: int
    group: str
    position: int
    points: int
    wins: int
    draws: int
    losses: int
    goals_for: int
    goals_against: int
